client/extensions/wp-super-cache/components/debug/index.jsx

```diff
@@ -49,6 +59,54 @@ class DebugTab extends Component {
 
 		return (
 			<div>
+				{ !! wp_super_cache_debug &&
+					<Card>
+						<p>
+							{ translate(
+								'Fix problems with the plugin by debugging it here. ' +
+								'It can log them to a file in your cache directory.'
+							) }
+						</p>
+						<FormFieldset>
+							<Button
+								compact
+								primary
+								disabled={ isRequesting || isSaving }
+								onClick={ this.resetLog }
+								value="1">
+								{ translate( 'Reset Debug Log' ) }
+							</Button>
+							<Button
+								compact
+								primary
+								disabled={ isRequesting || isSaving }
+								onClick={ this.disableLog }
+								value="1">
+								{ translate( 'Disable Debug Log' ) }
+							</Button>
+							<Button
+								compact
+								primary
+								disabled={ isRequesting || isSaving }
+								onClick={ this.deleteLog }
+								value="1">
+								{ translate( 'Disable and Delete Debug Log' ) }
+							</Button>
+						</FormFieldset>
+						<p>
+							{ translate(
+								'Currently logging to: %(location)s',
+								{ args: { location: cache_path + wp_cache_debug_log } }
+							) }
+						</p>
+						<p>
+							{ translate(
+								'Username and Password: %(username)s',
```

Comment:
Hi! I've found a possible matching string that has already been translated **82** times:
`translate( '[%s] Your username and password' )` *ES Score: 17*
See 1 additional suggestion in the [PR translation status page](https://translate.wordpress.com/localci/status/automattic/wp-calypso/add/wpsc-debug-log-buttons)

Help me improve these suggestions: react with :thumbsdown: if the suggestion doesn't make any sense, or with :thumbsup: if it's a particularly good one (even if not implemented).